config/initializers/omniauth.rb

```diff
@@ -1,10 +1,9 @@
-Rails.application.config.middleware.use OmniAuth::Builder do
-  provider(
-    :saml,
+SAML_SETTINGS = {
     assertion_consumer_service_binding: 'urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST',
     issuer: Rails.application.secrets.saml_issuer,
     idp_sso_target_url: Rails.application.secrets.idp_url,
-    idp_cert_fingerprint: '8B:D5:C2:E8:9A:2B:CE:B7:4B:95:50:BA:16:79:05:27:17:D1:D3:67',
+    idp_slo_target_url: Rails.application.secrets.idp_url.gsub(/auth$/, 'logout'),
```

Comment:
agreed, and I do that for the -dashboard config. I was trying to avoid needing to keep 2 different values in sync. I will re-visit.